cocotb/scheduler.py

```diff
@@ -505,6 +511,8 @@ def unschedule(self, coro):
                 outcome = outcomes.Error(e).without_frames(['unschedule', 'get'])
                 self._test._force_outcome(outcome)
             except Exception as e:
+                if _pdb_on_exception:
+                    pdb.post_mortem(e.__traceback__)
```

Comment:
This won't work on Python 2

Reply:
I modified it so it logs a warning if the exception doesn't have a '__traceback__' attribute.